<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LikeWindow</class>
 <widget class="QWidget" name="LikeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_user_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_userid">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_access_token">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Access token from FB API Explorer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>GET your posts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_windowinfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Info window</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_window">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>501</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LikeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Like</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DislikeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dislike</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalcount">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total count
your posts</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_count_post">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_token">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_count_liked">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalcount_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Count liked/
disliked</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_windowinfo_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>cUrl command from Mozilla through the Firebug</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="qText_cUrlFromMozilla">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
